### DISCUSSION TITLE
How to contribute to the project?

### DISCUSSION THREAD

**Discussion by Tipigao** [gfxfundamentals/webgl2-fundamentals#220] [General]
Hello!
My name is Roberto, I'm from Brazil.
I would like to contribute to the Brazilian version of this documentation.
Could you include me in the project to contribute translations into my language?

My user in git: *Tipigao*

**Comment by greggman** [comment]
In general you just start making [pull requests](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests) with Brazilian translations in the [`webgl/lessons/pt-br` folder](https://github.com/gfxfundamentals/webgl2-fundamentals/tree/master/webgl/lessons/pt-br)

There are some terse instructions here: https://github.com/gfxfundamentals/webgl2-fundamentals?tab=readme-ov-file#translating


You might also consider contributing to [webgpufundamentals](https://github.com/webgpu/webgpufundamentals)

**Reply by Tipigao** [reply]
I am not allowed to contribute, I need to upload a branch to make the Pull Request.
This would be the procedure, right?

The error message on execute _git push_:
`Permission to gfxfundamentals/webgl2-fundamentals.git denied to Tipigao.`

**Comment by Tipigao** [comment]
Thank you very much for the instructions.
I have already made a small correction in one of the documents, and will
soon make a Pull Request.
I will also analyze the webgpufundamentals [webgpufundamentals](https://github.com/webgpu/webgpufundamentals) project to contribute later.

**Comment by greggman** [comment]
To contribute, you fork the project, make a branch, make a change, upload it to your own fork, then create a pull request 

https://docs.github.com/pt/get-started/exploring-projects-on-github/contributing-to-a-project

**Reply by Tipigao** [reply]
Got it, I'll go this way
